tensornetwork/network.py

```diff
@@ -1206,8 +1206,8 @@ def split_node_full_svd(
     return left_node, singular_values_node, right_node, trun_vals
 
   def remove_node(self, node: network_components.BaseNode
-                 ) -> Tuple[Dict[Text, network_components.
-                                 Edge], Dict[int, network_components.Edge]]:
+                 ) -> Tuple[Dict[Text, network_components
```

Comment:
Please fix.

Reply:
Sorry about these formatting issues. I'm using py-yapf on emacs which makes these weird formatting decisions sometimes. I haven't figured out how to change it yet, and manual changes are a pain. Can we postpone this for a little longer? Rather have the bug fixed as soon as possible